ui/components/app/transaction-list/transaction-list.component.js

```diff
@@ -499,7 +506,13 @@ export default function TransactionList({
     setSelectedTransaction(transaction);
   }, []);
 
-  const multichainNetwork = useMultichainSelector(
+  const multichainNetwork = useSelector(
```

Comment:
Nit: Could have been named `multichainNetworkConfig` (so that would have matched the `networkConfig=` prop down below).